privacyidea/lib/crypto.py

```diff
@@ -107,16 +108,18 @@ def hmac_digest(self, data_input, hash_algo):
         self._clearKey_(preserve=self.preserve)
         return h
 
-    def aes_decrypt(self, data_input):
+    def aes_ecb_decrypt(self, enc_data):
         '''
-        support inplace aes decryption for the yubikey
+        support inplace aes decryption for the yubikey (mode ECB)
 
-        :param data_input: data, that should be decrypted
+        :param enc_data: data, that should be decrypted
```

Comment:
If I understand correctly, the length of ``enc_data`` should be a multiple of 16 bytes. Hm, should we maybe add this to the docstring?

Reply:
will do.

Reply:
actually this is the decrypt routine for yubikeys so i think we can assume that we get properly encrypted and padded data here.